docs/installation.rst

```diff
@@ -1,31 +1,42 @@
 Installation
 ============
 
-via pipx
+As tool
 --------
 
-:pypi:`tox` is a CLI tool that needs a Python interpreter (version 3.7 or higher) to run. We recommend :pypi:`pipx` to
-install tox into an isolated environment. This has the added benefit that later you'll be able to upgrade tox without
-affecting other parts of the system.
+:pypi:`tox` is a CLI tool that needs a Python interpreter (version 3.7 or higher) to run. We recommend either
+:pypi:`pipx` or  :pypi:`ruff` to install tox into an isolated environment. This has the added benefit that later you'll
```

Comment:
is this a typo? should it be uv and not ruff?

Reply:
Ah yes!